applications/neuroscan/frontend/src/components/viewers/viewer.js

```diff
@@ -140,7 +140,7 @@ class Viewer extends React.Component {
 
         const instances = fetchedNeurons.map((neuron) => mapToInstance(neuron));
 
-        addInstancesToViewer(viewerId, instances);
+        addInstancesToViewer(viewerId, instances, timePoint);
```

Comment:
question: @afonsobspinto the viewer is connected to a timepoint, why did you add `timePoint` as parameter?

Reply:
If the widget / viewer exists already the timepoint is not used.

For the situations where the viewer doesn't exist yet, like you said we need to connect it to a timepoint. 

The previous codebase was using the timepoint stored in the redux store to do that. I don't believe that is the correct behavior for situations where I want to add instances to a new viewer from a viewer in a different timepoint than the one currently in store:


https://github.com/MetaCell/geppetto-NeuroSCAN/assets/19196034/1b6c1114-ba5a-4ca5-9736-b348c84776ec